iceoryx_hoofs/test/moduletests/test_utility_convert.cpp

```diff
@@ -644,107 +683,105 @@ TEST_F(convert_test, fromString_Float_EdgeCase_InRange_Success)
 {
     ::testing::Test::RecordProperty("TEST_ID", "cf849d5d-d0ed-4447-89b8-d6b9f47287c7");
 
-    std::string source = fp_to_string(std::numeric_limits<float>::min());
+    // the number larger than numeric_limits<long double>::digits10 that will pass all tests for all platforms
+    constexpr uint16_t PLATFORM_DIGIT_WORKAROUND_MIN{7};
+    constexpr uint16_t PLATFORM_DIGIT_WORKAROUND_MAX{7};
+
+    std::string source = fp_to_string(std::numeric_limits<float>::min(), PLATFORM_DIGIT_WORKAROUND_MIN);
     auto float_min = iox::convert::from_string<float>(source.c_str());
     ASSERT_THAT(float_min.has_value(), Eq(true));
     EXPECT_THAT(float_min.value(), FloatEq(std::numeric_limits<float>::min()));
 
-    source = fp_to_string(std::numeric_limits<float>::lowest());
+    source = fp_to_string(std::numeric_limits<float>::lowest(), PLATFORM_DIGIT_WORKAROUND_MAX);
     auto float_lowest = iox::convert::from_string<float>(source.c_str());
     ASSERT_THAT(float_lowest.has_value(), Eq(true));
     EXPECT_THAT(float_lowest.value(), FloatEq(std::numeric_limits<float>::lowest()));
 
-    source = fp_to_string(std::numeric_limits<float>::max());
+    source = fp_to_string(std::numeric_limits<float>::max(), PLATFORM_DIGIT_WORKAROUND_MAX);
     auto float_max = iox::convert::from_string<float>(source.c_str());
     ASSERT_THAT(float_max.has_value(), Eq(true));
     EXPECT_THAT(float_max.value(), FloatEq(std::numeric_limits<float>::max()));
 }
 
-TEST_F(convert_test, fromString_Float_EdgeCase_SubNormalFloat_ShouldFail)
+TEST_F(convert_test, fromString_Float_EdgeCase_SubNormalFloat_ShouldFailExceptMsvc)
```

Comment:
```suggestion
TEST_F(convert_test, fromString_Float_EdgeCase_SubNormalFloat_ShouldFail
```
Can you please also change it for the other cases with `ExceptMsvc`

Reply:
I don't notice that :<

Reply:
No problem :)